redact/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@arcjet/redact",
+  "version": "1.0.0-alpha.21",
+  "description": "Arcjet PII Redaction Library",
+  "license": "Apache-2.0",
+  "homepage": "https://arcjet.com",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arcjet/arcjet-js.git",
+    "directory": "arcjet-redact"
```

```suggestion
    "directory": "redact"
```
You renamed the directory